pom.xml

```diff
@@ -96,8 +96,7 @@
         <h2.version>2.3.230</h2.version>
         <hibernate-orm.plugin.version>6.2.13.Final</hibernate-orm.plugin.version>
         <hibernate.c3p0.version>6.2.13.Final</hibernate.c3p0.version>
-        <infinispan.version>15.0.12.Final</infinispan.version>
-        <protostream.plugin.version>5.0.12.Final</protostream.plugin.version>
+        <infinispan.version>15.0.13.Final</infinispan.version>
```

Comment:
@ryanemerson - so this is now aligned in the versioning of Infinispan?

Reply:
BTW, there actually seems to be a version 15.0.13.Final published on Jan 23

https://mvnrepository.com/artifact/org.infinispan.protostream/proto-schema-compatibility-maven-plugin

Reply:
Unfortunately we accidentally released Protostream 15.0.13 instead of 5.0.13 on the firs try.